package.json

```diff
@@ -55,13 +56,15 @@
     "cpx": "^1.5.0",
     "eslint": "^5.15.2",
     "jest": "^24.5.0",
+    "ng-packagr": "^5.7.1",
     "rimraf": "^2.6.2",
     "single-spa": "^4.3.5",
     "ts-jest": "^24.0.0",
     "typescript": "3.2.4",
     "webpack": "^4.29.6"
   },
   "dependencies": {
+    "@angular/common": "^7.2.0",
```

Comment:
We should delete this because we don't want people to have to use Angular 7 when using single-spa-angular. Adding this as a dependency makes all users of single-spa-angular download this library at a specific version even though we don't want that.

The ng-packagr stuff you've added already adds angular/common as a peer dependency

Reply:
I changed it to be compiled with ngc